pages/instructorAssessmentQuestionStatistics/instructorAssessmentQuestionStatistics.ejs

```diff
@@ -60,7 +60,7 @@
           </div>
         </div>
 
-        <% if (assessment_stat.number > 0) { %>
+        <% if (assessment.score_stat_number > 0) { %>
```

Comment:
It's not clear to me why this is part of the conditional - isn't this unrelated to question stats?

Reply:
Sort of. As mentioned above, this page is using the _assessment_ statistics value `score_stat_number` (the number of students who have taken the assessment) as a flag to decide whether to show any _question_ statistics. I'm definitely not defending this choice but I'm also trying to make the minimal necessary changes to the question statistics pages in this PR.